### PR TITLE
[7.0] Remove node 0.10, 0.12 and 5 from Travis

### PR DESCRIPTION
**Pull request by chicoxyzzy**
| Q                 | A
| ----------------- | ---
| Bug fix?          | no
| Breaking change?  | yes
| New feature?      | no
| Deprecations?     | yes
| Spec compliancy?  | no
| Tests added/pass? | no
| Fixed tickets     | #282 
| License           | MIT

### CODE REVIEW

**Comment by chicoxyzzy** [comment]
Possibly we also should add [engines](https://docs.npmjs.com/files/package.json#engines) field to package.json

**Comment by danez** [comment]
Thank you. 

Yes good catch with package.json. Can you add this?

File: package.json [hunk 1]
```diff
@@ -11,6 +11,10 @@
     "bin",
     "lib"
   ],
+  "engines" : {
+    "node" : ">=4.2.0 <5.0.0 || >=6.9.0",
+    "npm": ">=2.14.7"
```
Comment (danez) [inline package.json]: I think we want to go with simply doing >4 for node, and we don't require any special npm version, so we also do not need to put a npm line here.

Comment (chicoxyzzy) [inline package.json]: npm 2.14.7 is a version which comes with Node 4 LTS Argon. Node 4 version proir to 4.2 [are not maintained](https://github.com/nodejs/LTS). So I suppose this should be than
```
{
  "node": ">=4.2",
  "npm": ">=2.14.7"
}
```

**Comment by danez** [comment]
Thank you very much.

**Pull request merged**